dom-events/buttons.js

```diff
@@ -0,0 +1,26 @@
+function handleClick(event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+}
+
+const test = document.querySelector('button.click-button');
```

Comment:
`test` isn't a really good name for a variable. You want your  variable names to be descriptive of what their value will be. Something like `$clickButton` would be better